src/css/components/_captions-settings.scss

```diff
@@ -67,6 +67,24 @@
   margin-right: 10px;
 }
 
+.vjs-caption-settings .hiddenReadable {
```

Comment:
Sure thing.  Was planning on using them for labels that could be read by screen readers but not displayed but haven't added any to the form.